contracts/flashMint/base/FlashLoanReceiverBase.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Is this file actually needed ? It is not used anywhere
IERC3156FlashBorrower should be sufficient